Tensile/SolutionStructs.py

```diff
@@ -968,6 +973,7 @@ def checkIfSupportedGEMMType(self):
 
     # TODO- Migrate ([H/H/H]+HPA) to ([H/H/S]+HPA)
     # Note that we need to do a little change in rocBLAS and logic yaml
+    # Currently, ([H/H/S]+HPA) is implemented, but due to some kenerl naming conflict, we use HBH kernels instead HHS_BH. 
```

Comment:
I think this PR accomplished the TODO, so you can just remove this whole block of comments (including the new one you added)

Reply:
This PR did not fully accomplish the TODO, as we eventually need to move HBH to HHS_BH to have a consistent naming convention. This has been mentioned in Chirs' comment down there (second bullet).